assets/scripts/dialogs/__tests__/GeotagDialog.test.js

```diff
@@ -17,10 +20,165 @@ describe('GeotagDialog', () => {
     expect(wrapper.exists()).toEqual(true)
   })
 
-  it('does not allow a location to be confirmed when geocoded data does not have street data')
-  it('allows a location to be confirmed when the current signed-in user is the street owner')
-  it('allows a location to be confirmed when the current anonymous user started this street')
-  it('does not allow a location to be confirmed when the current signed-in user is not the street owner')
-  it('does not allow a location to be confirmed when the current anonymous user is not the street owner and there is already an existing location attached')
-  it('allows a location to be confirmed when the current anonymous user is not the street owner but there is no existing location attached')
+  it('does not allow a location to be confirmed when geocoded data does not have street data', () => {
+    const testMarker = { lat: 0, lng: 0 }
+    const testAddressInfo = {
+      street: null
+    }
+    const wrapper = shallow(
+      <GeotagDialog.WrappedComponent
+        street={{}}
+        addressInformation={testAddressInfo}
+        markerLocation={testMarker}
+      />
+    )
+    expect(wrapper.find('button .confirm-button')).toHaveLength(0)
+  })
+  it('allows a location to be confirmed when the current signed-in user is the street owner', () => {
+    const testMarker = { lat: 0, lng: 0 }
+    const testAddressInfo = {
+      street: 'test street',
```

Comment:
It's more common to use [metasyntactic variable names](https://en.wikipedia.org/wiki/Metasyntactic_variable) like "foo" and "bar" as placeholder test data, so let's do that for consistency.